src/index.js

```diff
@@ -0,0 +1,195 @@
+const cityUpdate = () => {
+  const newCityText = state.cityTextField.value;
+  state.headerCity.textContent = newCityText;
+};
+
+const cityDefault = "Boston";
+
+const cityReset = () => {
+  state.cityTextField.value = cityDefault;
+  state.headerCity.textContent = state.cityTextField.value;
+};
+
+const landscapes = {
+  coldLandscape: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+  coolLandscape: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+  warmLandscape: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+  hotLandscape: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+};
+
+const skyOptions = {
+  sunny: "☁️ ☀️ ☁️ ☀️ ☀️  ☁️",
+  cloudy: "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+  rainy: "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+  snowy: "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨",
+};
+
+const landscapeUpdate = () => {
+  let newLandscape;
+  if (state.currentTemperature >= 80) {
+    newLandscape = landscapes.hotLandscape;
+  } else if (state.currentTemperature >= 70) {
+    newLandscape = landscapes.warmLandscape;
+  } else if (state.currentTemperature >= 60) {
+    newLandscape = landscapes.coolLandscape;
+  } else {
+    newLandscape = landscapes.coldLandscape;
+  }
+
+  state.landscapeField.textContent = newLandscape;
+};
+
+//--- wave 5
+const skyOptionsUpdate = () => {
+  let skyValue = state.skyOptions.value;
+  if (skyValue === "2") {
+    state.skyField.textContent = skyOptions.sunny;
+  } else if (skyValue === "3") {
+    state.skyField.textContent = skyOptions.cloudy;
+  } else if (skyValue === "4") {
+    state.skyField.textContent = skyOptions.rainy;
+  } else if (skyValue === "5") {
+    state.skyField.textContent = skyOptions.snowy;
+  } else {
+    state.skyField.textContent = "";
+  }
+};
+
+const weatherOptions = {
+  Clouds: "3",
+  Haze: "3",
+  Mist: "3",
+  Clear: "2",
+  Rain: "4",
+  Drizzle: "4",
+  Thunderstorm: "4",
+  Snow: "5",
+};
+
+const state = {
+  cityTextField: null,
+  headerCity: null,
+  cityResetButton: null,
+  landscapeField: null,
+  skyField: null,
+  skyOptions: null,
+  currTempButton: null,
+  temperatureValue: null,
+  currentTemperature: 50,
```

Comment:
As with the `currentTemperature` being used to track the temperature without being tied directly to a part of the UI, consider including variables for the selected sky and city name rather than using the `value` properties on their input controls directly. It's often useful to separate the internal value for something we're tracking from a particular input control, making it more flexible in how we set/calculate the values.